static/app/utils/replays/replayReader.tsx

```diff
@@ -252,17 +251,18 @@ export default class ReplayReader {
 
   getChapterFrames = memoize(() =>
     [
-      ...this._sortedBreadcrumbFrames.filter(
-        frame =>
-          [
-            'replay.init',
-            'ui.click',
-            'replay.mutations',
-            'ui.slowClickDetected',
-          ].includes(frame.category) || !BreadcrumbCategories.includes(frame.category)
```

Comment:
including `!BreadcrumbCategories.includes(frame.category)` means that all the custom crumbs are included.

These should be part of the console instead.